conf/usm_example.hosts

```diff
@@ -14,3 +14,6 @@ example-usm1-server.usmqe.tendrl.org
 
 [usm_client]
 example-usm1-client.usmqe.tendrl.org
+
+[tendrl_server]
+example-usm1-server.usmqe.tendrl.org
```

Comment:
Could you define the `usm_server` role so that it references to the already defined `tendrl_server` one? Like it's done for gluster usm node grous:

```
[usm_server:children]
tendrl_server
```